plugins/auth-backend/src/lib/PassportStrategyHelper.ts

```diff
@@ -58,6 +59,40 @@ export const makeProfileInfo = (
   };
 };
 
+export const makeOAuth2ProfileInfo = (
```

Comment:
Can't we just reuse the existing `makeProfileInfo`? The only difference should be that provider ends up being undefined. Would be fine with forcing typescript to like the situation for now, there's lots of usage of `any` and things to refactor around the profile handling anyway.

Reply:
I decided to create a new type here for if there were any use cases where custom actions needed to be performed on the params. Such as passing organization-specific values in the params that need to be accounted for and handled. There may be a better place in the code to handle that logic but I saw the `makeProfileInfo` as the best palce.  In the open-source project, it may be best to just use  `makeProfileInfo` and add a comment that this is where you would handle the params; if it is indeed the best place to do so.